src/OpenTelemetry.Exporter.OneCollector/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Dropped support for the `net7.0` target because .NET 7 is no longer supported.
+  Adds `net8.0` target.
+  ([#2038](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/2038))
```

Comment:
```suggestion
* Dropped support for the `net7.0` target because .NET 7 is no longer supported.
  Added a `net8.0` target.
  ([#2038](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/2038))
```